src/components/Rating/Rating.e2e.ts

```diff
@@ -0,0 +1,27 @@
+import { browser } from 'protractor';
```

Comment:
I would add a test that hovers over the component, and tests that the correct number of stars are "hovered"